superset-frontend/src/explore/components/controls/AnnotationLayerControl/AnnotationLayer.jsx

```diff
@@ -385,11 +385,13 @@ export default class AnnotationLayer extends React.PureComponent {
         description = 'Select the Annotation Layer you would like to use.';
       } else {
         label = t('Chart');
-        description = `Use a pre defined Superset Chart as a source for annotations and overlays.
-        your chart must be one of these visualization types:
-        [${this.getSupportedSourceTypes(annotationType)
-          .map(x => x.label)
-          .join(', ')}]`;
+        description = t(
+          `Use another existing chart as a source for annotations and overlays.
+          Your chart must be one of these visualization types: [%s]`,
+          this.getSupportedSourceTypes(annotationType)
+            .map(x => x.label)
+            .join(', '),
+        );
```

Comment:
If this seems to convoluted, you could do it like this:
```suggestion
        const supportedTypes = this.getSupportedSourceTypes(annotationType)
            .map(x => x.label)
            .join(', '),
        );
        const supportedTypesInstructions = t(
         `Use another existing chart as a source for annotations and overlays.
          Your chart must be one of these visualization types`
        );
        description = `${supportedTypesInstructions}: [{$supportedTypes}]`
```
... but either way works! ¯\\\_(ツ)_/¯

Reply:
Picked the previous one, to keep consistency. 

Thanks. 👍